src/script/media/MediaDevicesHandler.js

```diff
@@ -251,7 +251,8 @@ export default class MediaDevicesHandler {
   }
 
   /**
-   * Update list of available Screens.
+   * TODO: Make it usable for chrome because currently it's a wrapper-only thing.
```

Comment:
This todo can be removed. Chrome should handle the screen selection popup itself, we don't need to hook into it